lib/checksizeof.cpp

```diff
@@ -404,7 +404,7 @@ void CheckSizeof::sizeofVoid()
         return;
 
     for (const Token *tok = mTokenizer->tokens(); tok; tok = tok->next()) {
-        if (Token::simpleMatch(tok, "sizeof ( )")) { // "sizeof(void)" gets simplified to sizeof ( )
+        if (Token::simpleMatch(tok, "sizeof ( )") || Token::simpleMatch(tok, "sizeof ( void )")) {
```

Comment:
Thanks.. now I guess we should skip the first test.. `if (Token::simpleMatch(tok, "sizeof ( void )")) {`

Reply:
Yes you are correct. This is now fixed in my latest commit.